features/dd-sdk-android-session-replay/src/test/kotlin/com/datadog/android/sessionreplay/internal/async/RecordedDataQueueHandlerTest.kt

```diff
@@ -359,6 +380,16 @@ internal class RecordedDataQueueHandlerTest {
 
         // Then
         assertThat(testedHandler.recordedDataQueue.size).isEqualTo(0)
```

Comment:
side note: we can use this PR to change `assertThat(testedHandler.recordedDataQueue.size).isEqualTo(0)` assertions in this file to `assertThat(testedHandler.recordedDataQueue).isEmpty`